ivy/functional/ivy/general.py

```diff
@@ -2076,14 +2078,29 @@ def print_all_arrays_in_memory():
 
 
 def set_queue_timeout(timeout):
-    """Set the global queue timeout values (in seconds). Default value without this
-    function being called is 10 seconds.
+    """
+    Set the global queue timeout value (in seconds) 
+    Default value without this function being called is 15 seconds.
 
     Parameters
     ----------
     timeout
-        The timeout to set in seconds.
+        The timeout when waiting for containers to arrive from the queues. To be set in seconds.
```

Comment:
The ` lint test` is failing because the line is > 88 chars. Can you please reformat it so that the tests pass?

Reply:
Yes all done. Let me know if there is anything I can do